openmdao/recorders/tests/test_sqlite_reader.py

```diff
@@ -2857,6 +2857,19 @@ def test_system_metadata_attribute_deprecated(self):
         with assert_warning(DeprecationWarning, msg):
             options = cr.system_metadata
 
+    def test_system_options_attribute_deprecated(self):
+        model = om.Group()
+        model.add_recorder(self.recorder)
+        prob = om.Problem(model)
+        prob.setup()
+        prob.run_model()
+        prob.cleanup()
+
+        cr = om.CaseReader(self.filename)
+        msg = "The system_options attribute is deprecated. Use `list_model_options` instead."
+        with assert_warning(DeprecationWarning, msg):
+            options = cr.system_options
```

Comment:
also need to document the new function, and remove the section for the deprecated attribute here:
http://openmdao.org/twodocs/versions/3.2.1/features/recording/reading_metadata.html?highlight=system_options